### PR TITLE
Updated My Profile

### PR DESCRIPTION
**Pull request by rupeshexe**
## Changes proposed
Changes On my Profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3786"></a>

### CODE REVIEW

File: data/rupeshexe.json [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
+    "Data Science"
+  ],
   "socials": [
     { "icon": "FaTwitter", "url": "https://twitter.com/rupeshtwt" },
     { "icon": "FaGithub", "url": "https://github.com/rupeshexe" },
-    {
-      "icon": "FaLinkedin",
-      "url": "https://linkedin.com/in/rupeshexe"
-    }
+    { "icon": "FaYoutube", "url": "https://youtube.com/@rupeshexe" }
```
Comment (kumarsonsoff3) [inline data/rupeshexe.json:18]: Please add the missing comma `,` here to validate your json.  

```suggestion
    { "icon": "FaYoutube", "url": "https://youtube.com/@rupeshexe" },
```

File: data/rupeshexe.json [hunk 2]
```diff
@@ -28,6 +33,49 @@
...
+    {
+      "name": "YouTube: First Video Soon",
+      "url": "https://youtube.com/@rupeshexe",
+      "icon": "FaYoutube",
+      "color": "#FF0000"
+    },
+    {
+      "name": "Open Source and Random Tech Blogs",
+      "url": "https://rupeshexe.hashnode.dev/",
+      "icon": "FaHashnode"
```
Comment (kumarsonsoff3) [inline data/rupeshexe.json:46]: Another comma `,` missing there. 

```suggestion
      "icon": "FaHashnode",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/rupeshexe.json [hunk 3]
```diff
@@ -28,6 +33,49 @@
...
+      "date": "January 2023",
+      "icon": "FaTwitter",
+      "color": "blue",
+      "description": "Recreated my Twitter Account and trying to stay active this time.",
+      "url": "https://twitter.com/rupeshtwt"
+    },
+    {
+      "title": "First Quality PR on Github",
+      "date": "January 2023",
+      "icon": "FaGitHub",
```
Comment (kumarsonsoff3) [inline data/rupeshexe.json:67]:
```suggestion
      "icon": "FaGithub",
```

File: data/rupeshexe.json [hunk 4]
```diff
@@ -28,6 +33,49 @@
...
+      "color": "green",
+      "description": "First PR merged in a live website. Contributed in HTML",
+      "url": "https://github.com/rupeshexe"
+    },
+    {
+      "title": "Started with Open-Source",
+      "date": "October 2021",
+      "icon": "FaGitHub",
```
Comment (kumarsonsoff3) [inline data/rupeshexe.json:75]:
```suggestion
      "icon": "FaGithub",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
2 more fixes, then it should be good to merge! 

Invalid icon name is used, it should be `FaGithub` in both the places.

File: data/rupeshexe.json [hunk 5]
```diff
@@ -28,6 +33,63 @@
...
+    },
+    {
+      "name": "GitHub Sponsors: Support my Work",
+      "url": "https://github.com/sponsors/rupeshexe",
+      "icon": "FaDollarSign"
+    },
+    {
+      "name": "Mail me at",
+      "url": "mailto:contactrupeshdubey@gmail.com",
+      "icon": "SiGmail"
```
Comment (kumarsonsoff3) [inline data/rupeshexe.json:57]: You missed a comma `,` again in the Mail link you just added. 

```suggestion
      "icon": "SiGmail",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Finally, we are good to goo! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**